lib/freq_estimator_impl.cc

```diff
@@ -33,7 +33,10 @@ namespace gr {
               d_old_sig_avg(0.0),
               d_decim(decim),
               d_counter(decim)
-    {}
+    {
+        std::cout << "freq_estimator_impl::freq_estimator_impl 2 " << std::endl;
```

Comment:
`std::cout` in production code?

Reply:
Sorry, meh I failed to properly separate my current debugging session from the MR (As well the stuff below is a mix from a different commit)